Tasks/PackerBuild/Strings/resources.resjson/zh-CN/resources.resjson

```diff
@@ -1,5 +1,9 @@
 {
+<<<<<<< HEAD
```

Comment:
this is a conflict which needs to be resolved. Also, if this is the same in releases/m122 then that also needs to be fixed at priority